de.dlr.sc.virsat.model.ui/src/de/dlr/sc/virsat/model/ui/provider/ActiveConceptsLabelProvider.java

```diff
@@ -25,10 +25,24 @@
 	public String getText(Object element) {
 		if (element instanceof ActiveConceptConfigurationElement) {
 			ActiveConceptConfigurationElement acce = (ActiveConceptConfigurationElement) element;
-			return acce.getId() + " [" + acce.getVersion() + "]";
+			String acceDisplayName = acce.getDisplayName();
+			
+			if (acceDisplayName == null) {
+				return acce.getId() + " [" + acce.getVersion() + "]";

```

Comment:
Maybe we can extract `acce.getId() + " [" + acce.getVersion() + "]"` from the two branches so we dont have the logic duplicated.

Reply:
@SaMuellerDLR What exactly do you mean by two branches? Nevertheless, i think the whole logic could be packed into the ActiveConceptConfigurationElement. WHat do you think? This method could than be considered for a test case as well.

Reply:
@PhilMFischer Thats also a good idea. I just meant that there seems to be duplicate logic in the if and the else part of the if-else statement.

Reply:
Yes i saw it later with the if else...